keylime/tenant.py

```diff
@@ -1171,22 +1171,28 @@ def do_verify(self) -> None:
                     continue
                 self.do_cvstop()
                 raise e
+
+            mac = ""
+            ex_mac = crypto.do_hmac(self.K, challenge)
+
             if response.status_code == 200:
```

Comment:
```
mac = response_json.get("results",{}).get("hmac")
if not mac:
    logger.critical(...
```

Reply:
In the original code, a response code other than `200` triggered the execution of `keylime_logging.log_http_response`, and I just elected to preserve it. Trying to make it as minimalistic as possible... this (`tenant`) code will require a more extensive overhaul in the future (we really need to revisit the whole protocol), and for now I just want to make sure we stop getting random (timing) errors when PRs are open.